spikingjelly/activation_based/neuron/ilif.py

```
@@ -28,163 +36,289 @@ def __init__(
+        :param v_threshold: 神经元阈值电压，必须为有限正数，默认为 1.0
         :type v_threshold: float
-        :param max_spike_count: 训练时单个逻辑步最多释放的整数计数 ``D``
-        :type max_spike_count: int
         :param decay: 充电过程中历史膜电位的衰减 ``beta``，必须位于 ``[0, 1]``。
-            当 ``decay=1`` 时退化为无泄漏的整数 IF 形式
+            当 ``decay=1`` 时退化为无泄漏的整数 IF 形式，默认为 0.25
         :type decay: float
-        :param detach_reset: 是否在反向传播时分离 reset 中的发放值
+        :param surrogate_function: 多级整数发放函数，必须是 ``spiking=True`` 的
+            :class:`~spikingjelly.activation_based.surrogate.MultiLevelSpikeCount`。
+            ``max_spike_count`` 和矩形替代梯度区间均由该对象配置。若为 ``None``，
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)